authz.go

```diff
@@ -6,15 +6,23 @@ package authz
 
 import (
 	"net/http"
+	"strings"
 
 	"github.com/casbin/casbin"
 	"github.com/gin-gonic/gin"
 )
 
+var customUsername *string
+
 // NewAuthorizer returns the authorizer, uses a Casbin enforcer as input
-func NewAuthorizer(e *casbin.Enforcer) gin.HandlerFunc {
+func NewAuthorizer(e *casbin.Enforcer, username ...string) gin.HandlerFunc {
```

Comment:
``NewAuthorizer()`` is used to initialize the Casbin authorizer and it is only executed once, usually when the Gin service startups. So a static username doesn't make sense, as there cannot be only one username for the whole service?